src/license/THIRD-PARTY.properties

```diff
@@ -0,0 +1,56 @@
+# Generated by org.codehaus.mojo.license.AddThirdPartyMojo
+#-------------------------------------------------------------------------------
+# Already used licenses in project :
+# - Apache-1.1
+# - Apache-2.0
+# - BSD
+# - BSD License 3
+# - BSD-2-Clause
+# - CDDL
+# - CDDL + GPLv2 with classpath exception
+# - CDDL 1.1
+# - CDDL License
+# - CDDL v1.1 / GPL v2 dual license
+# - CDDL-1.0
+# - CPL-1.0
+# - Dual license consisting of the CDDL v1.1 and GPL v2
+# - EDL 1.0
+# - EPL-1.0
+# - EPL-2.0
+# - GPL2 w/ CPE
+# - GPLv2+CE
+# - Indiana University Extreme! Lab Software License, vesion 1.1.1
+# - LGPL-2.0
+# - LGPL-2.1
+# - Lesser General Public License, version 3 or greater
+# - MIT
+# - MPL-1.1
+# - MPL-2.0
+# - Modified BSD
+# - Public Domain
+# - Public Domain, per Creative Commons CC0
+# - The GNU General Public License (GPL), Version 2, With Classpath Exception
+# - The PostgreSQL License
+# - W3C license
+# - jQuery license
+#-------------------------------------------------------------------------------
+# Please fill the missing licenses for dependencies :
+#
+#
+#Wed Mar 03 10:03:56 CET 2021
+
+# https://github.com/javaee/jaxb-v2/blob/jaxb-2_1-branch/jaxb-ri/build.xml
+javax.xml.bind--jaxb-api--2.1=Dual license: COMMON DEVELOPMENT AND DISTRIBUTION LICENSE (CDDL) Version 1.0 + GNU General Public License (GPL) Version 2 with Classpath Exception
```

Comment:
As it turned out missed licenses just needs to be specified here in *.properties file just against the missed ones. Good fix, thank you for filling them. I thought the issue had to be solved somehow other.